src/main/java/org/ibp/api/rest/inventory/manager/LotResource.java

```diff
@@ -125,6 +131,17 @@
 		@PathVariable final String cropName,
 		@RequestParam(required = false) final String programUUID,
 		@RequestBody final LotsSearchDto lotsSearchDto) {
+
+		if (lotsSearchDto.getStudyId() != null) {
```

Comment:
Put a comment on this code to explain the reason why we need to get all gids from studyId.